### PR TITLE
Added a run command on build to drone

### PR DESCRIPTION
**Pull request by TheNeikos**

### CODE REVIEW

**Comment by TheNeikos** [comment]
btw, this fails because waysome segfaults before and thus pkill exits with 1 instead of 0

File: .drone.yml [hunk 1]
```diff
@@ -6,3 +6,6 @@ script:
     - cmake -DHARD_MODE=ON .
     - make
     - ctest -VV
+    - WAYSOME_LOG_LEVEL=8
+    - src/waysome &
+    - sleep 10s && pkill waysome
```
Comment (matthiasbeyer) [inline .drone.yml:11]: Why not `pidof waysome && pkill waysome` ?

File: .drone.yml [hunk 2]
```diff
@@ -6,3 +6,6 @@ script:
     - cmake -DHARD_MODE=ON .
     - make
     - ctest -VV
+    - WAYSOME_LOG_LEVEL=8
+    - src/waysome &
```
Comment (matthiasbeyer) [inline .drone.yml:10]: Does it actually work to put the env var and the call on two separate lines?

File: .drone.yml (hunk 1 above)
Comment (matthiasbeyer) [inline .drone.yml:11]: Btw: I would not sleep _10_ seconds! That's way too long to wait for!

**Comment by neithernut** [comment]
We triggered the test after `waysome`'s connection manager got fixed, but the test kept failing.
Is it possible that `waysome` dies because there is no DRM device?

File: .drone.yml (hunk 1 above)
Comment (neithernut) [inline .drone.yml:11]: Killing `waysome` before it even had the possibility to segfault is excactely _not_ what we want here.

**Comment by neithernut** [comment]
Mh... now we have the problem of waysome saying it can't initialize the input module...
Maybe `/dev/input` doesn't exist or is empty?

**Pull request closed without merging**